supplementary/alica_ros1/alica_tests/include/alica_timer.h

```diff
@@ -0,0 +1,112 @@
+#pragma once
```

Comment:
Not fully reviewing yet, but I think it's fine for the default alica timer to (for now) be in alica_engine rather than alica_tests.  Later we will have a package specifically for all default implementations